.gitignore

```diff
@@ -101,3 +101,6 @@ ENV/
 
 # patches
 *.patch
+
+# Sphinx
+docs/source/reference/generated/
```

Comment:
The warning read `No newline at end of file`.
Please insert a new line at end of the file.